package.json

```
@@ -28,7 +28,9 @@
     "package-lin": "./node_modules/.bin/electron-packager . --asar --overwrite  --out=./build --ignore=\"installers|tests|.vscode|coverage\" --platform=linux --arch=x64 --prune=true --icon=content/icons/png/256x256.png --executableName=threatdragon",
     "package-win": "./node_modules/.bin/electron-packager . --asar --overwrite  --out=./build --ignore=\"installers|tests|.vscode|coverage\" --platform=win32 --arch=x64 --prune=true --icon=content/icons/icon.ico --win32metadata.CompanyName=OWASP --win32metadata.FileDescription=\"OWASP Threat Dragon\" --win32metadata.ProductName=\"OWASP Threat Dragon\"",
     "snyk-protect": "snyk protect",
-    "prepare": "npm run-script snyk-protect"
+    "prepare": "npm run-script snyk-protect",
+    "help": "./node_modules/.bin/electron . help",
+    "pdf": "./node_modules/.bin/electron . pdf"
```

The CLI commands are working properly when using the executable (after building the project). However, when using npm just like you did with 'help' and 'pdf', maybe it could be useful to also add it for edit, open and print:
```suggestion
    "pdf": "./node_modules/.bin/electron . pdf",
    "edit": "./node_modules/.bin/electron . edit",
    "open": "./node_modules/.bin/electron . open",
    "print": "./node_modules/.bin/electron . print"
```

---

Otherwise the functionality is seamless!